ansible/config_sonic_basedon_testbed.yml

```diff
@@ -523,9 +523,20 @@
             path: /etc/sonic/port_config.json
             state: absent
 
-      - name: execute cli "config load_minigraph -y" to apply new minigraph
+      - name: Copy dhcp_server config
+        copy: src=golden_config_db/dhcp_server_mx.json
+              dest=/tmp/dhcp_server.json
         become: true
-        shell: config load_minigraph -y
+        when: topo == "mx"
+
+      - name: Generate golden_config_db.json
+        generate_golden_config_db:
+          topo_name: "{{ topo }}"
+        become: true
+
+      - name: execute cli "config load_minigraph --override_config -y" to apply new minigraph
+        become: true
+        shell: config load_minigraph --override_config -y
```

Comment:
Is this command exactly same with `config load_minigraph -y` when no `golden_config_db.json` exists?

Reply:
If no `golden_config_db.json` exists, the command `config load_minigraph --override_config -y` will report error .
From config_db point of view, an empty `{}` inside golden config will generate the same config as `config load_minigraph -y`

Reply:
As Jingwen said, cli would return error. Hence in new ansible module, will generate an empty golden_config "{}"